speaking-portal-project/readme.md

```diff
@@ -0,0 +1,28 @@
+# Scripts
```

Comment:
It might be worth adding pre-setup install instructions, or we should add them to the main readme.
For example:
1. Install [Node.js version 19.0.0](https://nodejs.org/en/)
2. From terminal, run `npm install -g nodemon` to install [Nodemon](https://nodemon.io/)
3. from terminal, run `npm install -g ts-node` to install [ts-node](https://typestrong.org/ts-node/)

Reply:
I think this might be better suited for the main readme!